ingest.py

```diff
@@ -34,6 +34,7 @@
 persist_directory = os.environ.get('PERSIST_DIRECTORY')
 source_directory = os.environ.get('SOURCE_DIRECTORY', 'source_documents')
 embeddings_model_name = os.environ.get('EMBEDDINGS_MODEL_NAME')
+is_gpu_enabled = os.environ.get('IS_GPU_ENABLED')
```

Comment:
Better make it boolean here and avoid the  `== "True"` comparison in the rest of the code

`is_gpu_enabled = (os.environ.get('IS_GPU_ENABLED', 'False').lower() == 'true')` 

Supports True or true as valid env var values for True